modules/alloy/src/main/ts/ephox/alloy/behaviour/coupling/CouplingApis.ts

```diff
@@ -1,10 +1,17 @@
+import { Optional } from '@ephox/katamari';
+
 import { AlloyComponent } from '../../api/component/ComponentApi';
 import { CouplingState, CouplingConfig } from './CouplingTypes';
 
 const getCoupled = (component: AlloyComponent, coupleConfig: CouplingConfig, coupleState: CouplingState, name: string): AlloyComponent =>
   // console.log('state', coupleState.readState());
   coupleState.getOrCreate(component, coupleConfig, name);
 
+// This will not create the coupled component if it does not exist!
```

Comment:
I can remove. I think I'm starting to err on the side of putting in more comments than less, because a lot of this code isn't changing very often (some hasn't changed since the first consolidated commit of 5.0), and it can help to communicate to new developers what the distinction between things is. But I agree ... this one is pretty useless.

- [ ] remove comment

Reply:
Updated: comment removed.